docker/build_whl.dockerfile

```diff
@@ -42,8 +42,6 @@ RUN yarn config set cache-folder /yarn_cache
 # Copy all files in this directory
 COPY . .
 
-VOLUME /kolibridist/  # for mounting the whl files into other docker containers
```

Comment:
Yes, this makes sense to remove.

This was part of an earlier design where the volume for the container that has the build assets was being mounted from another container doing other packaging steps. we seem to have moved away from that design and instead use the `-v $$PWD/dist:/kolibridist` volume mount with assets showing up in /dist
https://github.com/learningequality/kolibri/blob/develop/Makefile#L221

Reply:
Yeah, i noticed the removal was inconsequential - curious about what the use case was for mounting to another container?

Reply:
From what I remember windows packageing (or another one) needed the .whl from the build-whl container (this was before the `-v $$PWD/dist:/kolibridist`)